File: app/src/main/java/com/vpe_soft/intime/intime/recyclerview/RecyclerViewCursorViewHolder.java
package com.vpe_soft.intime.intime.recyclerview;

import android.database.Cursor;
import android.view.View;
import androidx.recyclerview.widget.RecyclerView;

public abstract class RecyclerViewCursorViewHolder extends RecyclerView.ViewHolder {

    public RecyclerViewCursorViewHolder(View view) { super(view); }

    public abstract void bindCursor(Cursor cursor);
}

File: intime/app/src/main/java/com/vpe_soft/intime/intime/recyclerview/RecyclerViewCursorViewHolder.kt
package com.vpe_soft.intime.intime.recyclerview

import android.database.Cursor
import android.view.View
import androidx.recyclerview.widget.RecyclerView

abstract class RecyclerViewCursorViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    abstract fun bindCursor(cursor: Cursor)
}